tests/discussions.test.js

```diff
@@ -0,0 +1,364 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const Discussions = require('../src/models/discussions');
+const Accounts = require('../src/models/accounts');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+let accessToken= null;
+let idDiscussion = null;
+
+describe("POST Tests Discussion",() =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
describe("POST Tests Discussion",() => {
```